src/gui_common/CustomPopupMenu.cs

```diff
@@ -9,11 +9,15 @@ public class CustomPopupMenu : TopLevelContainer
     [Export]
     public NodePath? PanelPath;
 
+    [Export]
+    public NodePath? ClipControlPath;
```

Comment:
Regarding this NodePath as this is not the first nodepath this should not be nullable (this should be `    public NodePath ClipControlPath = null!;` like ContainerPath is below this). Also if I saw correctly a dispose call is missing for this.

Reply:
Yeah, forgot about dispose call. Getting used to how C# does things. Fixed now.

Reply:
All good. I was more surprised to see the CI check not detecting that undisposed field for some reason... I have no clue why it didn't pick up on that, the jetbrains analyzer should have caught it.

Reply:
Not sure either. Didn't catch it when I ran the code checker locally either.